### PR TITLE
Avoid calling Test-ServiceFabricApplicationPackage in case of upgrading without any change in service version 

### PR DESCRIPTION
**Pull request by kuvinodms**

### CODE REVIEW

File: Tasks/ServiceFabricDeployV1/deploy.ps1 [hunk 1]
```diff
@@ -107,7 +107,12 @@ try {
 
         if (!$skipValidation)
         {
-            $isPackageValid = Test-ServiceFabricApplicationPackage -ApplicationPackagePath $applicationPackagePath
+            $ApplicationManifestPath = Get-ApplicationManifestPath -ApplicationPackagePath $applicationPackagePath
```
Comment (bishal-pdMSFT) [inline Tasks/ServiceFabricDeployV1/deploy.ps1]: this change is not required

File: Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1 [hunk 2]
```diff
@@ -146,22 +146,41 @@ function Publish-UpgradedServiceFabricApplication
...
+    $names = Get-NamesFromApplicationManifest -ApplicationManifestPath $ApplicationManifestPath
+    if (!$names)
+    {
+        return
+    }
+    $oldApplication = Get-ServiceFabricApplication -ApplicationName $ApplicationName
     if (!$SkipPackageValidation)
     {
-        $packageValidationSuccess = (Test-ServiceFabricApplicationPackage $AppPkgPathToUse -ImageStoreConnectionString $imageStoreConnectionString)
+        $packageValidationSuccess = $true
```
Comment (bishal-pdMSFT) [inline Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1]: no change is required inside this if

File: Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1 [hunk 3]
```diff
@@ -146,22 +146,41 @@ function Publish-UpgradedServiceFabricApplication
...
     $imageStoreConnectionString = Get-ImageStoreConnectionStringFromClusterManifest ([xml] $clusterManifestText)
-
+    $ApplicationManifestPath = "$AppPkgPathToUse\ApplicationManifest.xml"
+    $names = Get-NamesFromApplicationManifest -ApplicationManifestPath $ApplicationManifestPath
+    if (!$names)
+    {
+        return
+    }
+    $oldApplication = Get-ServiceFabricApplication -ApplicationName $ApplicationName
     if (!$SkipPackageValidation)
```
Comment (bishal-pdMSFT) [inline Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1]: move skipUpgrade logic to here

**Comment by kuvinodms** [comment]
Closing this PR because of wrong target branch in this issue. Raised the new PR #7373 
@bishal-pdMSFT please have a look to the new PR

**Pull request closed without merging**